common/src/thrift/tachyon.thrift

```diff
@@ -36,12 +36,10 @@ struct BlockInfo {
 }
 
 // Information about files.
-// TODO: Just include a BlockInfo in this FileBlockInfo
 struct FileBlockInfo {
-  1: i64 blockId
+  1: BlockInfo blockInfo
   2: i64 offset
-  3: i64 length
-  4: list<NetAddress> locations
+  3: list<NetAddress> underFsLocations
```

Comment:
Could you please add a comment that explains the relationship between BlockInfo and FileBlockInfo and the difference between BlockInfo.locations and FileBlockInfo.underFsLocations. I think that this Thrift file should be well documented as it is the ground truth for the RPC interface of our services.